NitroxServer/GameLogic/Bases/BaseManager.cs

```diff
@@ -83,10 +138,13 @@ public void BasePieceConstructionCompleted(NitroxId id, NitroxId baseId)
 
                     partiallyConstructedPiecesById.Remove(id);
 
+                    //this task is now completed by BasePieceConstructionAmountChanged()
+                    /*
                     lock (completedBasePieceHistory)
                     {
                         completedBasePieceHistory.Add(basePiece);
                     }
+                    */
```

Comment:
We don't recommend leaving dead code in PRs.  We expect PRs to be merge worthy unless marked as "DO NOT MERGE"

Reply:
I will mark it as do not merge now, once I figure out how